
import logging
logger = logging.getLogger("MAIN")
logger.error("error happend in the app")
